build/gulp/tasks/docs.ts

```diff
@@ -58,8 +58,8 @@ const componentsSrc = [`${config.paths.src()}/components/*/[A-Z]*.tsx`]
 const examplesSrc = `${paths.docsSrc()}/examples/*/*/*/index.tsx`
 const markdownSrc = ['.github/CONTRIBUTING.md', 'specifications/*.md']
 
-task('build:docs:docgen', () =>
-  src(componentsSrc, { since: lastRun('build:docs:docgen') })
+task('build:docs:component-info', () =>
+  src(componentsSrc, { since: lastRun('build:docs:component-info') })
```

Comment:
The corresponding clean task for this is called `clean:component-info`, the util function is called `getComponentInfo`, I've then renamed the build task for consistency.

The legacy name `docgen` through me off when trying to use it.